moshi/src/main/java/com/squareup/moshi/-MoshiKotlinExtensions.kt

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (C) 2019 Square, Inc.
+ * Copyright (C) 2020 Square, Inc.
```

Comment:
FWIW we typically never bump these after creation

Reply:
9cd0535